opentelemetry-otlp/examples/basic-otlp/otel-collector-config.yaml

```diff
@@ -1,36 +1,27 @@
+# This is a configuration file for the OpenTelemetry Collector intended to be
+# used in conjunction with the opentelemetry-otlp example.
+#
+# For more information about the OpenTelemetry Collector see:
+#   https://github.com/open-telemetry/opentelemetry-collector
+#
 receivers:
   otlp:
     protocols:
-      http:
       grpc:
+      http:
 
 exporters:
   logging:
     loglevel: debug
 
-  jaeger:
```

Comment:
this was cleaned up as the purpose of this example is to show how to use OTLP Exporter. The config will print the telemetry to stdout which is good enough to confirm things working. Even if there is a need to use Jaeger to visualize, Jaeger can natively understand OTLP, so no need to futher export from collector.